website_product_show_uom/tests/test_website_uom_logout.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © 2016 Oscar Alcalá <oscar@vauxoo.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import openerp.tests
+
+
+@openerp.tests.common.at_install(False)
+@openerp.tests.common.post_install(True)
+class TestUi(openerp.tests.HttpCase):
+    def test_01_public_uom(self):
+        self.phantom_js(
+            "/", "openerp.Tour.run('website_test_uom_logout', 'test')",
+            "openerp.Tour.tours.website_test_uom_logout")
```

Comment:
Thanks a lot @andhit-r ,
I don't understand why you are creating another test for a not logged in user, the previous test `website_test_uom_login` can be used with no login is as simple as adding an extra test method like this:
```python
    def test_03_public_uom(self):
        self.phantom_js(
            "/", "openerp.Tour.run('website_test_uom_login', 'test')",
            "openerp.Tour.tours.website_test_uom_login")
```
just removing the `login` param as you did on this test should work.